.codesandbox/tasks.json

```
@@ -1,13 +1,13 @@
 {
     "setupTasks": [
       {
-        "name": "Startup",
-        "command": "npm install, npm start"
+        "name": "npm installation",
+        "command": "npm install" // I'm dumb, I know
```

dont need the comment in the file haha, it'll be confusing to see in the future

```suggestion
        "command": "npm install"
```